tests/fixtures/simple/next.config.js

```diff
@@ -8,6 +8,7 @@ const nextConfig = {
     outputFileTracingIncludes: {
       '/': ['public/**'],
     },
+    after: true,
```

Comment:
TODO: this test need to be moved to separate / new fixture because `experimental.after` and `(unstable_)after` import only works for next@>=15 and this test is used also with older versions

The test setup and test itself won't really change other than that and can be already reviewed

Reply:
- [ ] move to separate test that only uses `next >= 15`

Reply:
Done in https://github.com/opennextjs/opennextjs-netlify/pull/2717/commits/819be98917f25a3d7a6c9d726dbc38d438dbc2bb